packages/vertexai/src/requests/request-helpers.ts

```diff
@@ -81,16 +86,17 @@ function assignRoleToPartsAndValidateSendMessageRequest(
   }
 
   if (hasUserContent && hasFunctionContent) {
-    throw ERROR_FACTORY.create(VertexError.INVALID_CONTENT, {
-      message:
-        'Within a single message, FunctionResponse cannot be mixed with other type of part in the request for sending chat message.'
-    });
+    throw new VertexAIError(
+      VertexAIErrorCode.INVALID_CONTENT,
+      'Within a single message, FunctionResponse cannot be mixed with other type of part in the request for sending chat message.'
```

Comment:
Sorry for the nits on text that you haven't changed, but now that I see it, we should probably capitalize Part here and Content in the next error.

Reply:
Done 👍